scripts/ci/list-missing-dependencies.sh

```diff
@@ -0,0 +1,84 @@
+#!/bin/bash
+# List dependencies missing from the domain lists.
+
+domains=("common" "connect" "mission" "mobile" "qa" "trends" "usability")
+
+dependencies_in_lists=()
```

Comment:
Nit: maybe I'd name it `dependencies_in_all_domain_lists`, to make it really clear. But it's not necessary.